File: angular-app/src/app/api/models/hall-dto.ts
/* tslint:disable */
/* eslint-disable */
import { HallBookingDto } from './hall-booking-dto';
export interface HallDto {
  bookings?: null | Array<HallBookingDto>;
  capacity?: number;
  hallId?: number;
  hallName?: null | string;
  location?: null | string;
  ownerId?: number;
  pricePerHour?: number;
}

File: WebApplication1/WebApplication1/DTO/Response/HallDto.cs
using System.Text.Json.Serialization;
using WebApplication1.Models;

namespace WebApplication1.DTO.Response
{
    public class HallDto
    {

        public int HallId { get; set; }
        public string HallName { get; set; }
        public string Location { get; set; }
        public int capacity { get; set; }
        public List<HallBookingDto>? Bookings { get; set; } = null;
        public double PricePerHour { get; set; }
        public int ownerId { get; set; }
        [JsonIgnore]
        public User? HallOwner { get; set; }
    }
}